File: 2.0/php/contacts/deleteDoNotContactsBySource.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->deleteDoNotContactsBySource();
        $request->getOperationConfig()->setPathParameters(array("source" => "My DNC List 1"));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();

File: 2.0/java/contacts/deleteDoNotContactsBySource.java
import com.callfire.api.client.CallfireClient;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api login", "api password");
        client.dncApi().deleteDncsFromSource("My DNC List 1");
    }
}
